tests/pyincore/models/test_dfr3curve.py

```diff
@@ -76,3 +94,45 @@ def test_calculate_limit_state_probability(fragility_set, hazard_values, args, e
     assert np.isclose(result["LS_0"], expected)
 
 
+@pytest.mark.parametrize("repair_set,args,expected", [
+    (get_repair_set("repairset.json"), {"repair_time": [3.5, 1.2]}, 2),
+    (get_repair_set("repairset.json"), {"repair_time": 80}, 0.9943516689414926),
+    (get_remote_repair_set("60edf9a4fc0f3a7af53a2194"), {"repair_time": [3.5, 1.2]}, 2)
+])
+def test_calculate_repair_rates(repair_set, args, expected):
+    result = repair_set.calculate_repair_rates(**args)
+    if type(result["PF_0"]) == numpy.ndarray:
+        assert len(result["PF_0"]) == expected
+    elif type(result["PF_0"]) == numpy.float64:
+        assert result["PF_0"] == expected
+    else:
+        assert False
+
+
+@pytest.mark.parametrize("repair_set,args,expected", [
+    (get_repair_set("repairset.json"), {"repair_time": [0.5, 0.2]}, 2),
+    (get_repair_set("repairset.json"), {"repair_time": 0.67}, 27.50466741611462)
+])
```

Comment:
The way you write the repair time expected is a little bit confused... 
2 is the length
 and 27.5xxx is the value
could you make them two different variables and evalute both len and value?

Reply:
agreed. I can change it to an array value (instead of len) even for the list